Java/JDI/jdi-cucumber-test-generator/pom.xml

```diff
@@ -75,6 +86,33 @@
             <version>7.0.47</version>
             <scope>provided</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <version>${fasterxml.version}</version>
```

Comment:
May be it is timne to introduce dependencyManagement section in parent pom and keep all versions there?